Dockerfile

```diff
@@ -4,8 +4,7 @@ LABEL maintainer="labs@idealista.com"
 
 EXPOSE 8089
 
-RUN apk add gcc libc-dev linux-headers --update-cache \
-    && rm -rf /var/cache/apk/*
+RUN apk add gcc libc-dev linux-headers --no-cache
```

Comment:
shouldn't cache be updated?

Reply:
with --no-cache flag always do a fetch before because it isn't use any local cache at all.